### PR TITLE
Ufanete's First Goodie

### PR DESCRIPTION
**Pull request by ufanete**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Returns the static text "ufanete is awesome and has successfully completed the DuckDuckHack Goodie tutorial!"

**What problem does your Instant Answer solve (Why is it better than organic links)?**
My first tutorial

**What is the data source for your Instant Answer? (Provide a link if possible)**
Static text

**Why did you choose this data source?**
Not applicable.

**Are there any other alternative (better) data sources?**
Not applicable.

**What are some example queries that trigger this Instant Answer?**
duckduckhack ufanete

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Geek

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
From a friend

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-03-09 at 6 11 25 pm

## Checklist
Please place an 'X' where appropriate.

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [X] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [X] Google Chrome
        - [X] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

File: t/IsAwesome/ufanete.t [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+use warnings;
+use Test::More;
+use DDG::Test::Goodie;
+
+zci answer_type => "is_awesome_ufanete";
+zci is_cached   => 1;
+
+ddg_goodie_test(
+    [qw( DDG::Goodie::IsAwesome::ufanete )],
+    # At a minimum, be sure to include tests for all:
```
Comment (moollaza) [inline t/IsAwesome/ufanete.t]: Could you please remove these comments? They're just for you to read and understand. No need to keep them in the source.

**Comment by javathunderman** [comment]
@ufanete LGTM. :cake: 

cc @moollaza if I missed anything

File: lib/DDG/Goodie/IsAwesome/ufanete.pm [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package DDG::Goodie::IsAwesome::ufanete;
+# ABSTRACT: ufanete's first Goodie
+
+use DDG::Goodie;
+
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/ufanete.pm]: Can you add `use strict;` here? It's not mandatory, but it keeps github from detecting the script as perl 6 by accident

Comment (javathunderman) [inline lib/DDG/Goodie/IsAwesome/ufanete.pm]: @mintsoft Thanks for picking that up, forgot about that...

**Comment by mintsoft** [comment]
Congratulations on your first goodie @ufanete . Thanks for the review @javathunderman

**Comment by moollaza** [comment]
@ufanete congrats! It's live: https://duckduckgo.com/?q=duckduckhack+ufanete&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**